src/google/protobuf/arenaz_sampler_test.cc

```diff
@@ -592,6 +592,9 @@ TEST(ThreadSafeArenazSamplerTest, UsedAndWasted) {
   });
   SetThreadSafeArenazSampleParameter(oldparam);
 }
+
+#include "google/protobuf/port_undef.inc"
```

Comment:
This should be at the bottom of the file

Reply:
my bad, got confused thinking this was an h file and it needing to be before the endif like any.h

#include "google/protobuf/port_undef.inc"

#endif  // GOOGLE_PROTOBUF_ANY_H__

Reply:
fixed and signed.

Reply:
Will look at the tests failing tomorrow as well as changing my signature email to match the commits